app/controllers/twilio_webhooks_controller.rb

```diff
@@ -3,7 +3,9 @@ class TwilioWebhooksController < ActionController::Base
   before_action :validate_twilio_request
 
   def update_outgoing_text_message
-    OutgoingTextMessage.find(params[:id]).update(twilio_status: params["MessageStatus"])
+    status = params["MessageStatus"]
+    DatadogApi.increment("twilio.outgoing_text_messages.updated.status.#{status}")
```

Comment:
dumb question - are these always nice lowercase under_scored_words strings? If they're not, might be nice to transform them.

Reply:
As far as I can tell, yes, they're always lower case single words.
https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-